src/test/java/teammates/test/cases/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -134,15 +137,16 @@ public void testExceptionalCases() throws Exception {
 
         resultsPage = loginToInstructorFeedbackResultsPage("CFResultsUiT.SanitizedTeam.instr",
                                                            "Session with sanitized data");
-
+        clickAjaxLoadedPanelAndWaitForExpansion("panelHeading-1", "ajax_auto");
         resultsPage.verifyHtmlMainContent("/instructorFeedbackResultsPageWithSanitizedData.html");
     }
 
     private void testModerateResponsesButton() {
 
         resultsPage = loginToInstructorFeedbackResultsPage("CFResultsUiT.instr", "Open Session");
         resultsPage.displayByQuestion();
-
+        clickAjaxLoadedPanelAndWaitForExpansion("panelHeading-2", "ajax_auto");
```

Comment:
I put it here to follow the convention within this method that environment set up is above _____TS and verifyModerateResponsesButton is below _____TS. But I think I should have added a new line here to avoid confusion!